packages/lesswrong/components/editor/EditorFormComponent.jsx

```diff
@@ -175,24 +189,76 @@ class EditorFormComponent extends Component {
 
   changeCount = 0
   setDraftJS = (value) => { // Takes in an editorstate
-    const { document, name } = this.props
     const { draftJSValue } = this.state
     const currentContent = draftJSValue.getCurrentContent()
     const newContent = value.getCurrentContent()
-    if (currentContent !== newContent) {
-      // Only save to localStorage on every 30th content change
-      // TODO: Consider debouncing rather than saving every 30th change
-      // TODO: Consider saving on blur
-      this.changeCount = this.changeCount + 1;
-      if (this.changeCount % 30 === 0) {
-        const rawContent = convertToRaw(newContent);
-        this.getStorageHandlers().set({state: rawContent, doc: document, name})
-      }
-    }
+    const changed = (currentContent != newContent);
     this.setState({draftJSValue: value})
+    
+    if (changed)
+      this.maybeSaveBackup();
```

Comment:
Make this one-line, or add curly braces, in this case I feel uneasy about the lack of braces